src/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -117,7 +117,9 @@ exports[`DataTable selection should have select-all default to un-checked if no
           <TableHead>
             <thead>
               <TableRow>
-                <tr>
+                <tr
+                  className=""
```

Comment:
Could we update the `classNames` usage so that we don't have an empty `className` generated in the snapshot?